getnacelle/contentful-app-nacelle
# feat: Nacelle Reference page and table

## Why are these changes introduced?

Addresses [ENG-9415](https://nacelle.atlassian.net/browse/ENG-9415) 

## What is this pull request doing?

- Adds a Nacelle Reference page to the Nacelle app
- Uses [CMA](https://www.contentful.com/developers/docs/references/content-management-api/) to fetch `contentTypes` and `EditorInterfaces`
- Adds test to ensure the table contains the appropriate data

## How to test

- `npm i && npm run test` -> all tests should pass
- Shape of the [`expected data`](https://github.com/getnacelle/contentful-app-nacelle/pull/19/files#diff-a51acca2034afc132e862e3118c9f5df56f60305983f908da14ba423dafe8c05R48) is in line with ADR075 

- `npm run start`
  - Should open a browser tab
  - Select `Prairie Wind Apparel` space
  - Select `Staging` environment
  - Click on the `Apps` menu and select `Nacelle [Local Dev]`


## Checklist



[ENG-9415]: https://nacelle.atlassian.net/browse/ENG-9415?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
